extensions/amp-inline-gallery/1.0/README.md

```diff
@@ -0,0 +1,319 @@
+
+[example preview="top-frame" playground="false"]
+
+Install via npm:
+
+```sh
+npm install @ampproject/bento-inline-gallery
+```
+
+```javascript
+import React from 'react';
+import { BentoInlineGallery } from '@ampproject/bento-inline-gallery/react';
+import '@ampproject/bento-inline-gallery/styles.css';
+
+function App() {
+  return (
+    <BentoInlineGallery id="inline-gallery">
+      <BentoInlineGalleryThumbnails aspect-ration="1.5" loop></BentoInlineGalleryThumbnails>
+      <BentoBaseCarousel snap-align="center" visible-count="1.2" loop>
+        <img src="server.com/static/inline-examples/images/image1.jpg">
+        <img src="server.com/static/inline-examples/images/image2.jpg">
+        <img src="server.com/static/inline-examples/images/image3.jpg">
+      </BentoBaseCarousel>
+      <BentoInlineGallery-pagination inset></BentoInlineGallery-pagination>
+    </BentoInlineGallery>
+  }
+```
+
+[/example]
+
+#### Interactivity and API usage
+
+Bento components are highly interactive through their API. The `BentoInlineGallery` component API is accessible by passing a `ref`:
+
+```javascript
+import React, {createRef} from 'react';
+const ref = createRef();
+
+function App() {
+  return (
+    <BentoInlineGallery ref={ref}>
+      <BentoInlineGallerySection id="section1" key={1}>
+        <BentoInlineGalleryHeader><h1>Section 1</h1></BentoInlineGalleryHeader>
+        <BentoInlineGalleryContent>Content 1</BentoInlineGalleryContent>
+      </BentoInlineGallerySection>
+
+      <BentoInlineGallerySection id="section2" key={2}>
+        <BentoInlineGalleryHeader><h1>Section 2</h1></BentoInlineGalleryHeader>
+        <BentoInlineGalleryContent>Content 2</BentoInlineGalleryContent>
+      </BentoInlineGallerySection>
+
+      <BentoInlineGallerySection id="section3" key={3}>
+        <BentoInlineGalleryHeader><h1>Section 3</h1></BentoInlineGalleryHeader>
+        <BentoInlineGalleryContent>Content 3</BentoInlineGalleryContent>
+      </BentoInlineGallerySection>
+    </BentoInlineGallery>
+  );
+}
+```
+
+##### Actions
+
+The `BentoInlineGallery` component does not surface an API, but instead requires that consumers interact with the [BentoBaseCarousel API](https://github.com/ampproject/amphtml/blob/main/extensions/amp-base-carousel/1.0/README.md#interactivity-and-api-usage-1) directly
+
+##### Events
+
+The `BentoInlineGallery` component does not surface an API, but instead requires that consumers interact with the [BentoBaseCarousel API](https://github.com/ampproject/amphtml/blob/main/extensions/amp-base-carousel/1.0/README.md#interactivity-and-api-usage-1) directly
+
+#### Layout and style
+
+**Container type**
+
+The `BentoInlineGallery` component has a defined layout size type. To ensure the component renders correctly, be sure to apply a size to the component and its immediate children via a desired CSS layout (such as one defined with `height`, `width`, `aspect-ratio`, or other such properties). These can be applied inline:
```

Comment:
I believe it just needs a defined `width` because its `height` comes from the defined `height` of its children.

Reply:
updated